Formula/carl-storm.rb

```diff
@@ -0,0 +1,39 @@
+class CarlStorm < Formula
+  desc "Computer ARithmetic and Logic library for the probabilistic model checker Storm"
+  homepage "https://ths-rwth.github.io/carl/"
+  url "https://github.com/moves-rwth/carl-storm/archive/refs/tags/14.25.tar.gz"
+  version "14.25"
+  sha256 "511740d53c2a6a41c3ccb3bd3b2d9ec89d0576f37b8804fc15fbe083e7a357da"
+
+  head "https://github.com/moves-rwth/carl-storm.git", using: :git
+
+  option "with-thread-safe", "Build with thread-safe support"
```

Comment:
I would favour enabling this by default. Carl-storm also sets it to true by default.